continuous_integration/push_doc.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# This script is meant to be called in the "deploy" step defined in 
+# circle.yml. See https://circleci.com/docs/ for more details.
+# The behavior of the script is controlled by environment variable defined
+# in the circle.yml in the top level folder of the project.
+
+
+if [ -z $CIRCLE_PROJECT_USERNAME ];
+then USR_NAME="sklearn-ci";
+else USR_NAME=$CIRCLE_PROJECT_USERNAME;
+fi
+
+if [ -z $CIRCLE_PROJECT_REPONAME ];
+then SRC_REPO="scikit-learn";
+else SRC_REPO=$CIRCLE_PROJECT_REPONAME;
+fi
```

Comment:
I put it here in case someone need to use this script locally to push manually the latest doc (since I assume Circle CI environment variable won't be available)